internal/process/hosts.go

```diff
@@ -50,7 +51,7 @@ func createHostSamples(config *load.Config, timestamp int64) {
 			if config.Args.DatacenterLocation != "" {
 				checkError(config, ms.SetMetric("datacenterLocation", config.Args.DatacenterLocation, metric.ATTRIBUTE))
 			}
-			checkError(config, ms.SetMetric("hypervisorHostName", hostConfigName, metric.ATTRIBUTE))
+			checkError(config, ms.SetMetric("hypervisorHostname", hostConfigName, metric.ATTRIBUTE))
```

Comment:
@paologallinaharbur let's sync this with the dashboard changes

Reply:
I update the dashboard, I will open the PR once we release these changes (maybe at the end of the beta)